### PR TITLE
feat: add chinese locale

### PR DESCRIPTION
**Pull request by ckvv**

### CODE REVIEW

**Comment by ckvv** [comment]
fix: #269

File: src/qComponents/constants/locales/zh.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    charNumber: '字符数'
+  },
+  QDatePicker: {
+    placeholder: '选择日期',
+    startPlaceholder: '开始',
+    endPlaceholder: '结束',
+    prevYear: '前一年',
+    nextYear: '后一年',
+    prevMonth: '前一月',
+    nextMonth: '前一月',
```
Comment (Tim152) [inline src/qComponents/constants/locales/zh.ts:22]: prevMonth / nextMonth are the same

**Review by Tim152 — DISMISSED** [review]

File: src/qComponents/constants/locales/zh.ts (hunk 1 above)
Comment (ckvv) [reply src/qComponents/constants/locales/zh.ts]: Fixed

Comment (Tim152) [reply src/qComponents/constants/locales/zh.ts]: great! thank you, I'll merge it for now to add some changes and test it in local machine

**Pull request merged**